server/app/views/components/Modal.java

```diff
@@ -34,7 +34,7 @@ private Modal(ModalBuilder builder) {
   public Tag getContainerTag() {
     return div()
         .withId(modalId)
-        .withClasses(ReferenceClasses.MODAL, BaseStyles.MODAL, width.getStyle())
```

Comment:
Since there could be other issues of this type, I'd suggest adjusting the script and rerunning it on a fresh copy (that way we don't have to think about reviewing the PR where some changes were automated and others were hand-crafted).

Reply:
Oh! You're right. I'll backtrack and adjust my script. Just heads up I gotta go in like 40min. Will be back in the evening on this

Reply:
I got that fixed

Reply:
sorry, I coulda sworn that was fixed now, but now it should be good